movies/.eslintrc.js

```diff
@@ -3,10 +3,7 @@ module.exports = {
     browser: true,
     es2020: true,
   },
-  extends: [
-    'plugin:react/recommended',
-    'airbnb',
-  ],
+  extends: ["airbnb", "plugin:react/recommended", "prettier", "prettier/react"],
```

Comment:
prettier and prettier/react should be specified in project devdependencies

Reply:
Fixed